hail/python/hail/docs/change_log.md

```diff
@@ -24,6 +24,29 @@ an earlier version of Hail to read files written in a later version.
 
 ---
 
+## Version 0.2.28
+
+Released 2019-11-22
+
+### Critical correctness bug fix
+- (hail#7588) Fixes a bug where filtering old matrix tables in newer versions of hail did not work as expected. Please update from 0.2.27. 
+
+### Bug fixes
+- (hail#7577) Fixed an optimizer bug.
```

Comment:
Can you add #7571 to the changelog also? I believe it's also included in this release?

Reply:
Done!

Reply:
Thanks!